pyoos/collectors/nerrs/nerrs_soap.py

```diff
@@ -30,6 +45,8 @@ def get_stations(self):
                 if val is None:
                     val = ""
                 s[child.tag] = val
+            s['Longitude'] = -float(s['Longitude'])
```

Comment:
We can still check if it is negative and then enforce it.  Don't know NERRS, but that kind of assumptions makes me uneasy :wink: